manim/cli/render/commands.py

```diff
@@ -103,6 +104,8 @@ def __repr__(self):
     if args["jupyter"]:
         return click_args
 
+    nov = config.notify_outdated_version
+
```

Comment:
I'm pretty sure you can just reference the config dict for the if condition after rendering on L42
```suggestion
```

Reply:
> I'm pretty sure you can just reference the config dict for the if condition after rendering on L142

`config.digest_args(click_args)` on L109 considers no arguments entered as `False` for `notify_outdated_version`

Reply:
Perhaps we should remove it as a flag and keep it as `True` by default? Then later on we can add it back if necessary.

Reply:
In practice, passing the CLI flag should overwrite the config files and passing no CLI flag shouldn't modify the ManimConfig object. Since it does, that's a bug, though I'm unsure if it's click/cloup related or ManimConfig related -- I'll checkout your branch and see what I can do to move this PR along; but I'd prefer to keep the flag around.

Reply:
I believe it has something to do with the changes to `manim/_config/utils.py`

Reply:
This works in global_options.py:
```
    option(
        "--notify_outdated_version/--silent",
        is_flag=True,
        default=None,
        help="Display warnings for outdated installation.",
    ),
```
So the flag takes 3 values, False/True if specified, otherwise None so that the config file can do its job